config/kernel-monitor.json

```diff
@@ -58,6 +63,12 @@
 			"reason": "IOError",
 			"pattern": "Buffer I/O error .*"
 		},
+		{
+			"type": "permanent",
+			"condition": "XfsShutdown",
+			"reason": "XfsHasShutdown",
+			"pattern": "XFS .* Shutting down filesystem.?"
```

Comment:
Should it end with `$`?

Reply:
It ends with `.` since Linux v5.15.

See https://github.com/torvalds/linux/commit/b36d4651e1650082d27fa477318183c4a7210e30#diff-45a1e8a0f0b13276e2f4fb066678c2fbd056f3bd47725c2ed6d612c9d3f7bc59R555